redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/event/CertReplaceInRedBeamsRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.event;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
+import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsEvent;
+
+public class CertReplaceInRedBeamsRequest extends RedbeamsEvent {
```

Comment:
Possibly rename to `CertRotateInRedbeamsRequest` (so `Replace` -> `Rotate` and `Beams` made all lowercase).